lms/djangoapps/support/urls.py

```diff
@@ -5,19 +5,16 @@
 
 from lms.djangoapps.support.views.contact_us import ContactUsView
 from support.views.certificate import CertificatesSupportView
-from support.views.course_entitlements import EntitlementSupportView
```

Comment:
Lets double check this, since I merged the base view/template this is actually pointing to the correct support view now, and shouldn't be removed

Reply:
Yeah, all the changes to this urls file should be removed from this PR.